aspnetcore/client-side/bundling-and-minification.md

```diff
@@ -61,10 +61,22 @@ The MVC and Razor Pages project templates provide an out-of-the-box solution for
 
 ## Configure bundling and minification
 
-The MVC and Razor Pages project templates provide a *bundleconfig.json* configuration file which defines the options for each bundle. By default, a single bundle configuration is defined for the custom JavaScript (*wwwroot/js/site.js*) and stylesheet (*wwwroot/css/site.css*) files:
+::: moniker range="<= aspnetcore-2.0"
+
+In ASP.NET Core 2.0 or earlier, the MVC and Razor Pages project templates provide a *bundleconfig.json* configuration file which defines the options for each bundle:
+
+::: moniker-end
+
+::: moniker range=">= aspnetcore-2.1"
+
+Add a new JSON file, named *bundleconfig.json*, to the MVC or Razor Pages project root. Include the following JSON in that file as a starting point:
```

Comment:
To be consistent can we start the sentence with the following?
In ASP.NET Core 2.1 or later, add a new JSON file, named ...